polyphemus/config.yml.template

```diff
@@ -1,7 +1,11 @@
 :development:
   :magma:
-    :host: https://magma.ucsf.edu
+    :host: https://magma.development.local
+  :mvir1:
+    :token: my-token # create a long lived token with janus
 
-:production:
+:test:
   :magma:
-    :host: https://magma.ucsf.edu
+    :host: https://magma.test
+  :mvir1:
+    :token: mvir1-token
```

Comment:
NB: I'm pretty un-excited about writing project names into config files any more and would much rather put this into a database column as soon as possible.